demo/project_demo12-HopsworksTikTokRecSys/features/interactions.py

```diff
@@ -0,0 +1,213 @@
+from mimesis import Generic
```

Comment:
Is this code borrowed from Hopsworks? If so, please add attribution to a module-level comment.

Reply:
Yes, this is borrowed from Hopsworks.